src/main/java/edu/cornell/kfs/coa/batch/businessobject/LaborClosedAccount.java

```diff
@@ -0,0 +1,85 @@
+package edu.cornell.kfs.coa.batch.businessobject;
+
+import java.sql.Date;
+import java.text.SimpleDateFormat;
+import java.util.Locale;
+
+import org.apache.commons.lang3.StringUtils;
+import org.kuali.kfs.sys.KFSConstants;
+
+public class LaborClosedAccount {
+    
+    private String chart;
+    private String accountNumber;
+    private String subAccountNumber;
+    private String accountClosedIndicator;
+    private Date accountClosedDate;
+    
+    public LaborClosedAccount() {
+        this.chart = null;
+        this.accountNumber = null;
+        this.subAccountNumber = null;
+        this.accountClosedIndicator = null;
+        this.accountClosedDate = null;
+    }
+    
+    public LaborClosedAccount(String chart, String  accountNumber, String subAccountNumber, String accountClosedIndicator, Date accountClosedDate) {
```

Comment:
I don't think this constructor is used, is it needed?

Reply:
Removed constructor.